### PR TITLE
[1.18] Add keybind for opening the mod menu screen.

### LINKED ISSUE #403: [Feature Request] A shortcut key to Mod menu
On some occasions, I need some quick mod configurating, or entering mod menu many times (for testing mod configs). So maybe adding a shortcut key to enter mod menu, just like most clients that have a shortcut key to the client settings, is a pretty efficient idea. :)

### PR DESCRIPTION
**Pull request by MagnusHJensen**
This includes the feature requested and closes #403

This is my first interaction with fabric, but hopefully not my last.
Feel free to comment on my approach, or the default keybind for the key.

Also I'm unsure how translations work for this specific project, so I'm not sure if I should add the translations keys somewhere (en file), or if they need to be added through Weblate.

If accepted, I'll be happy to port to 1.16 and 1.19 :D

### CODE REVIEW

File: src/main/java/com/terraformersmc/modmenu/event/ModMenuEventHandler.java [hunk 1]
```diff
@@ -6,24 +6,37 @@
...
 
 public class ModMenuEventHandler {
 	private static final Identifier FABRIC_ICON_BUTTON_LOCATION = new Identifier(ModMenu.MOD_ID, "textures/gui/mods_button.png");
+	private static KeyBinding MENU_KEY_BIND;
 
 	public static void register() {
+		MENU_KEY_BIND = KeyBindingHelper.registerKeyBinding(new KeyBinding(
+				"key.modmenu.open_menu",
+				InputUtil.Type.KEYSYM,
+				GLFW.GLFW_KEY_R,
```
Comment (jackassmc) [inline src/main/java/com/terraformersmc/modmenu/event/ModMenuEventHandler.java]: "M" would make sense but all the map mods use it already. Maybe leave it unbound by default?

**Review by jackassmc — COMMENTED** [review]
Looks great overall, please do add the strings to en_us.json, it is the translation source file that gets synced to Weblate.

**Comment by haykam821** [comment]
A client command would be appreciated personally, but I understand that it's outside of the scope of this pull request.

**Comment by MagnusHJensen** [comment]
> A client command would be appreciated personally, but I understand that it's outside of the scope of this pull request.

How would a command to pull up the menu benefit one?

File: src/main/resources/assets/modmenu/lang/en_us.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
+  "category.modmenu.name": "Mod Menu",
+  "key.modmenu.open_menu": "Opens the Mod Menu screen.",
```
Comment (jackassmc) [inline src/main/resources/assets/modmenu/lang/en_us.json]:
```suggestion
  "key.modmenu.open_menu": "Open Mod Menu",
```

**Comment by jackassmc** [comment]
Please also port to 1.17.

**Comment by jackassmc** [comment]
Thank you!

**Pull request merged**